wazimap_np/templates/homepage.html

```diff
@@ -77,25 +77,27 @@ <h2 class="column-full subhead">Exploring data</h2>
+                <p>NepalMap uses <a href="http://nationaldata.gov.np/" target="_blank">National Data Profile</a>, created by <a href="http://www.cbs.gov.np/" target="_blank">Central Bureau of Statistics</a>, and other official sources, to create user-friendly data visualizations on key demographic issues.</p>
+                <p>You can <span class="bold">embed, access or download</span> the data too. You can also compare any two areas side-by-side.</p>
+                <p>NepalMap is a project of <a href="http://codefornepal.org/en/" target="_blank">Code for Nepal</a>, a non-profit that works to increase digital literacy and the use of open data in Nepal. Please make a <a href="https://www.paypal.com/fundraiser/charity/117735" target="_blank">tax deductible donation</a> to help us maintain the server space for NepalMap, and empower Nepal digitally!</p>
+            <!-- </p> -->
 <h2 class="column-full subhead">Shoulders of Giants</h2>
-<p>Yes! NepalMap is based on <a href="https://github.com/Code4SA/wazimap" target="_blank">Wazimap</a> which is a fork of <a href="https://censusreporter.org/" target="_blank">Census Reporter</a>. Wazimap is built and maintained by <a href="http://code4sa.org/" target="_blank">Code for South Africa</a>. Other implementations include <a href="https://kenya.wazimap.org/" target="_blank">Wazimap Kenya</a> and <a href="https://wazimap.co.za" target="_blank">Wazimap South Africa</a>, the first Wazimap implementation. All of these tools are completely open source. You can build <a href="https://wazimap.readthedocs.org/en/latest/index.html" target="_blank">one</a> for your country. NepalMap code is licensed under the MIT License.<p>
+<p>Yes! NepalMap is based on <a href="https://github.com/Code4SA/wazimap" target="_blank">Wazimap</a> which is 
+    a fork of <a href="https://censusreporter.org/" target="_blank">Census Reporter</a>. Wazimap is built and maintained by <a href="http://code4sa.org/" target="_blank">Code for South Africa</a>. Other implementations include <a href="https://kenya.wazimap.org/" target="_blank">Wazimap Kenya</a> and <a href="https://wazimap.co.za" target="_blank">Wazimap South Africa</a>, the first Wazimap implementation. All of these tools are completely open source. You can build <a href="https://wazimap.readthedocs.org/en/latest/index.html">one</a> for your country. NepalMap code is licensed under the MIT License.
+<p>
 
 <p>NepalMap branding is copyright of Code for Nepal and cannot be used without permission.</p> 
 
 <h2 class="column-full subhead">Help us</h2>
-<p>We are looking for volunteers who can help us add more data, and improve NepalMap. If you have the right skills, and would like to help, please complete this <a href="http://codefornepal.org/en/help/" target="_blank">form</a>! 
+<p>Code for Nepal is entirely built by volunteers who are committed to ensuring everyone has access to open data about Nepal. We need more help. </p>
```

Comment:
Would you consider adding a link to https://github.com/Code4Nepal/nepalmap_federal/graphs/contributors for "volunteers"?

```html
<p>Code for Nepal is entirely built by <a href="https://github.com/Code4Nepal/nepalmap_federal/graphs/contributors" target="_blank">volunteers</a> who are committed to ensuring everyone has access to open data about Nepal. We need more help. </p>
```

Reply:
@ravinepal 

You marked this as "resolved". Was that a way to signal agreement or to signal that we should **not** include a link to the contributors to the project?

Reply:
@ravinepal 

Bumping this comment about adding a link to the GitHub contributors here.